lib/services/rediscachemanagement/package.json

```diff
@@ -1,37 +1,25 @@
 {
   "name": "azure-arm-rediscache",
   "author": "Microsoft Corporation",
-  "contributors": [
-    "Thaniyulla Parambath, Alfan <alfantp@microsoft.com>",
-    "Dacosta, Carlyle <cadaco@microsoft.com>",
-    "Zavery, Amar <amzavery@microsoft.com>"
-  ],
+  "description": "RedisManagementClient Library with typescript type definitions for node",
   "version": "2.1.0-preview",
```

Comment:
Need to bump the version (I'm guessing 3.0.0).

Reply:
Yes, I'm waiting on publishing of autorest.nodejs so I can use the official package to regenerate.